include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1123,6 +1127,11 @@ namespace internal
                   lexicographic[no][fe_index].end(), lex.begin(), lex.end());
               }
 
+            identity[no][fe_index].resize(lexicographic[no][fe_index].size());
+            for (unsigned int i = 0; i < lexicographic[no][fe_index].size();
+                 ++i)
+              identity[no][fe_index][i] = i;
+
```

Comment:
Can't we skip this variable, skip the `lexicographic_inv` argument in `DoFInfo` altogether, and simply set up the renumbering now also in the multigrid code branch? It would slightly increase the code size compared to the old state, but now that we need to apply the transformation in one of the two places already, I see no reason to pass around arrays any more.